roofit/roofitcore/inc/RunContextTracker.h

```diff
@@ -0,0 +1,43 @@
+// Author: Jonas Rembser, CERN  Feb 2021
+
+/*****************************************************************************
+ * RooFit
+ * Authors:                                                                  *
+ *   WV, Wouter Verkerke, UC Santa Barbara, verkerke@slac.stanford.edu       *
+ *   DK, David Kirkby,    UC Irvine,         dkirkby@uci.edu                 *
+ *                                                                           *
+ * Copyright (c) 2000-2020, Regents of the University of California          *
+ *                          and Stanford University. All rights reserved.    *
+ *                                                                           *
+ * Redistribution and use in source and binary forms,                        *
+ * with or without modification, are permitted according to the terms        *
+ * listed in LICENSE (http://roofit.sourceforge.net/license.txt)             *
+ *****************************************************************************/
+
+#ifndef roofit_roofitcore_RunContextTracker_h
+#define roofit_roofitcore_RunContextTracker_h
+
+#include "RooChangeTracker.h"
+#include "RunContext.h"
```

Comment:
Replace by forward decls. Both are only used via `*` and `&`.

Reply:
`RooAbsArg/Real` is also not declared.

Reply:
Done.